ui/round/src/crazy/crazyCtrl.ts

```diff
@@ -43,7 +43,7 @@ export function valid(data: RoundData, role: Role, key: Key): boolean {
 
   const drops = dropStr.match(/.{2}/g);
 
-  return drops?.includes(key) === true;
+  return !!drops?.includes(key);
```

Comment:
`!!` is not needed for booleans.

Reply:
if `drops` is undefined/null `.includes` gives `undefined` right?